src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Storage interface that deals with the loading and storing of the tasks data.
+ */
+public class Storage {
+    /**
+     * The file path where the tasks data are stored.
+     */
+    private final String FILE_PATH;
```

Comment:
Should the naming be in this format? Consider changing to private static final.

Reply:
I shall use filePath then since duke is defining the file path